### DISCUSSION TITLE
FYI, we're no longer recommending /usr/etc/ as the place for your config files in your custom image of atomic Fedora

### DISCUSSION THREAD

**Discussion by xynydev** [blue-build/.github#35] [Announcements]
https://blue-build.org/blog/preferring-system-etc/

**Comment by gadgethm** [comment]
May also want to update https://blue-build.org/learn/mindset/ to reflect this change

**Reply by xynydev** [reply]
Good heads-up, this is an oversight on my part. Oops! Will update.
